spring-pulsar-spring-cloud-stream-binder/src/test/java/org/springframework/pulsar/spring/cloud/stream/binder/PulsarBinderUtilsTests.java

```diff
@@ -25,6 +25,9 @@
 import org.springframework.cloud.stream.provisioning.ConsumerDestination;
 import org.springframework.pulsar.spring.cloud.stream.binder.properties.PulsarConsumerProperties;
 
+/**
+ * @author Soby Chacko
```

Comment:
**[NIT]** Do you mind adding the boilerplate ` Unit tests for {@link PulsarBinderUtils}.` description?